[2019-06-15 12:06:53,010] WARN Client session timed out, have not heard from server in 5777ms for sessionid 0x1002a9c5ec40002 (org.apache.zookeeper.ClientCnxn)
[2019-06-15 12:06:53,220] INFO Client session timed out, have not heard from server in 5777ms for sessionid 0x1002a9c5ec40002, closing socket connection and attempting reconnect (org.apache.zookeeper.ClientCnxn)
[2019-06-15 12:06:55,197] INFO Opening socket connection to server localhost/0:0:0:0:0:0:0:1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-06-15 12:06:55,754] INFO Socket connection established to localhost/0:0:0:0:0:0:0:1:2181, initiating session (org.apache.zookeeper.ClientCnxn)
[2019-06-15 12:06:56,154] WARN Unable to reconnect to ZooKeeper service, session 0x1002a9c5ec40002 has expired (org.apache.zookeeper.ClientCnxn)
[2019-06-15 12:06:56,156] INFO [ZooKeeperClient] Session expired. (kafka.zookeeper.ZooKeeperClient)
[2019-06-15 12:06:56,155] INFO EventThread shut down for session: 0x1002a9c5ec40002 (org.apache.zookeeper.ClientCnxn)
[2019-06-15 12:06:56,173] INFO Unable to reconnect to ZooKeeper service, session 0x1002a9c5ec40002 has expired, closing socket connection (org.apache.zookeeper.ClientCnxn)
[2019-06-15 12:06:56,218] INFO [ZooKeeperClient] Initializing a new session to localhost:2181. (kafka.zookeeper.ZooKeeperClient)
[2019-06-15 12:06:56,222] INFO Initiating client connection, connectString=localhost:2181 sessionTimeout=6000 watcher=kafka.zookeeper.ZooKeeperClient$ZooKeeperClientWatcher$@55cb6996 (org.apache.zookeeper.ZooKeeper)
[2019-06-15 12:06:56,237] INFO Opening socket connection to server localhost/127.0.0.1:2181. Will not attempt to authenticate using SASL (unknown error) (org.apache.zookeeper.ClientCnxn)
[2019-06-15 12:06:56,495] INFO Socket connection established to localhost/127.0.0.1:2181, initiating session (org.apache.zookeeper.ClientCnxn)
[2019-06-15 12:06:56,571] INFO Session establishment complete on server localhost/127.0.0.1:2181, sessionid = 0x1002a9c5ec40003, negotiated timeout = 6000 (org.apache.zookeeper.ClientCnxn)
[2019-06-15 12:06:56,605] INFO Creating /brokers/ids/0 (is it secure? false) (kafka.zk.KafkaZkClient)
[2019-06-15 12:06:56,626] INFO Stat of the created znode at /brokers/ids/0 is: 6718,6718,1560618416608,1560618416608,1,0,0,72104445131096067,194,0,6718
 (kafka.zk.KafkaZkClient)
[2019-06-15 12:06:56,627] INFO Registered broker 0 at path /brokers/ids/0 with addresses: ArrayBuffer(EndPoint(10.161.56.13,9092,ListenerName(PLAINTEXT),PLAINTEXT)), czxid (broker epoch): 6718 (kafka.zk.KafkaZkClient)
[2019-06-15 12:06:56,678] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 10.161.56.13:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:278)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:256)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:920)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:287)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:282)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:236)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2019-06-15 12:06:56,805] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 10.161.56.13:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:278)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:256)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:920)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:287)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:282)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:236)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2019-06-15 12:06:56,948] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 10.161.56.13:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:278)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:256)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:920)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:287)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:282)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:236)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2019-06-15 12:06:57,073] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 10.161.56.13:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:278)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:256)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:920)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:287)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:282)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:236)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2019-06-15 12:06:57,184] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 10.161.56.13:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:278)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:256)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:920)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:287)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:282)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:236)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2019-06-15 12:06:57,286] WARN [Controller id=0, targetBrokerId=0] Error connecting to node 10.161.56.13:9092 (id: 0 rack: null) (org.apache.kafka.clients.NetworkClient)
java.net.SocketException: Network is unreachable
	at java.base/sun.nio.ch.Net.connect0(Native Method)
	at java.base/sun.nio.ch.Net.connect(Net.java:476)
	at java.base/sun.nio.ch.Net.connect(Net.java:468)
	at java.base/sun.nio.ch.SocketChannelImpl.connect(SocketChannelImpl.java:694)
	at org.apache.kafka.common.network.Selector.doConnect(Selector.java:278)
	at org.apache.kafka.common.network.Selector.connect(Selector.java:256)
	at org.apache.kafka.clients.NetworkClient.initiateConnect(NetworkClient.java:920)
	at org.apache.kafka.clients.NetworkClient.ready(NetworkClient.java:287)
	at org.apache.kafka.clients.NetworkClientUtils.awaitReady(NetworkClientUtils.java:65)
	at kafka.controller.RequestSendThread.brokerReady(ControllerChannelManager.scala:282)
	at kafka.controller.RequestSendThread.doWork(ControllerChannelManager.scala:236)
	at kafka.utils.ShutdownableThread.run(ShutdownableThread.scala:82)
[2019-06-15 12:10:47,894] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2019-06-15 12:20:47,898] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2019-06-15 12:30:47,943] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2019-06-15 12:40:47,936] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
[2019-06-15 12:50:47,933] INFO [GroupMetadataManager brokerId=0] Removed 0 expired offsets in 0 milliseconds. (kafka.coordinator.group.GroupMetadataManager)
